R/install.R

```diff
@@ -236,7 +236,7 @@ find_exec = function(cmd, dir, version = NULL, info = '') {
       uninstall_tip(path2), "."
     )
   }
-  normalizePath(path)
+  xfun::normalize_path(path)
```

Comment:
I change that because on Windows the path where not equivalent if you set `version = "all"` or anything else - because of the backslash. It seems not desired but I may have mistaken. Maybe the backslash are important ? 

```r
> blogdown::find_hugo("all")
[1] "C:/Users/chris/AppData/Roaming/Hugo/0.76.5/hugo.exe" "C:/Users/chris/AppData/Roaming/Hugo/0.78.2/hugo.exe"
[3] "C:/Users/chris/scoop/shims/hugo.exe"                
[<no git>] > blogdown::find_hugo()

==> Found hugo at "C:\Users\chris\AppData\Roaming/Hugo/0.78.2/hugo.exe" and "C:\Users\chris\scoop\shims\hugo.exe". The former will be used. If you don't need both copies, you may delete/uninstall the latter perhaps with system2("scoop", c("uninstall", "hugo")).

[1] "C:\\Users\\chris\\AppData\\Roaming\\Hugo\\0.78.2\\hugo.exe"
```

Reply:
Okay, let's keep this change.